commands/publish/__tests__/publish-canary-no-git-reset.test.js

```diff
@@ -73,24 +70,5 @@ test("publish --canary --no-git-reset", async () => {
   );
   await lernaPublish(cwd)("--canary", "--no-git-reset");
 
-  expect(PromptUtilities.confirm).toHaveBeenLastCalledWith(
-    "Are you sure you want to publish these packages?"
-  );
-  expect(npmPublish.registry).toMatchInlineSnapshot(`
-Map {
-  "package-1" => "canary",
-  "package-3" => "canary",
-  "package-4" => "canary",
-  "package-2" => "canary",
-}
-`);
-  expect(writePkg.updatedVersions()).toMatchInlineSnapshot(`
-Object {
-  "package-1": 1.0.1-alpha.0+SHA,
-  "package-2": 1.0.1-alpha.0+SHA,
-  "package-3": 1.0.1-alpha.0+SHA,
-  "package-4": 1.0.1-alpha.0+SHA,
-}
-`);
-  expect(gitCheckout).toHaveBeenCalledTimes(0);
+  expect(gitCheckout).not.toHaveBeenCalled();
```

Comment:
The majority of these assertions were removed because they were completely redundant and obscured the actual point of the test case.